styles/globals.scss

```diff
@@ -0,0 +1,17 @@
+html,
+body {
+    padding: 0;
+    margin: 0;
+    font-family: $font--primary;
+    background: $color--primary;
+    color: $color--white;
+}
+
+a {
+    color: #3beccd;
```

Comment:
**Suggestion**: Consider colour variable refactor here

Reply:
I wanted to do that, but I didn't know how to name it...🙂

Reply:
See above colour conversation on theming